src/components/TodoList.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import Todo from "./Todo";
+const TodoList = ({ todos, setTodos, filteredTodos }) => {
+  return (
+    <div className="todo-container">
+      <ul className="todo-list">
+        {filteredTodos.map((todo) => (
+          <Todo
+            setTodos={setTodos}
+            todos={todos}
+            key={todo.id}
+            todo={todo}
+            text={todo.text}
+          />
+        ))}
+      </ul>
+    </div>
+  );
+};
+
+export default TodoList;
+
+// linijka 8- usunął id={todo.id} i poxniej zmienil key={todo.id} i dał text={todo.text}
```

Comment:
komentarz do usunięcia,  jak używasz pętli jak tutaj, metoda "map()" to trzeba używać 'key' żeby react mógł wiedzieć który element z tablicy ma odświerzyć w przyszłości